spec/support/purl_helpers.rb

```diff
@@ -10,22 +11,10 @@ module PurlHelpers
   #  (Cocina)
   #    - access data ... with the embargo in it as appropriate
   #    - access data correctly put in PURL xml
-  #
-  # ideally, would look for the following on purl page:
-  #   "Access is restricted until #{embargo_date.strftime('%d-%b-%Y')}"
-  # but this is in the embed *file* viewer only and it's within an iframe and I couldn't figure it out.
-
   def expect_embargo_date_in_purl(druid, embargo_date)
-    Timeout.timeout(Settings.timeouts.publish) do
-      loop do
-        visit "#{Settings.purl_url}/#{druid}.xml"
-        break if html.match?('rightsMetadata')
-
-        sleep 1
-      end
-    end
-
-    purl_ng_xml = Nokogiri::XML(html)
+    Dor::Services::Client.configure(url: Settings.dor_services.url, token: Settings.dor_services.token)
```

Comment:
should there be a comment for the module or the method that clarifies this is not looking at the purl, but is looking at the public_xml that will be sent to the purl?   Or maybe even a module rename?

Reply:
I'll add a comment and rename. thanks!